src/ocm/components/clusterConfiguration/OcmDiscoveryImageConfigForm.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import * as Yup from 'yup';
+import {
+  Button,
+  Form,
+  ModalBoxBody,
+  ModalBoxFooter,
+  AlertVariant,
+  Alert,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { Formik, FormikHelpers } from 'formik';
+import { ImageCreateParams, ImageType, Proxy } from '../../../common/api';
+import {
+  AlertFormikError,
+  httpProxyValidationSchema,
+  noProxyValidationSchema,
+  sshPublicKeyValidationSchema,
+} from '../../../common/components/ui';
+import { ProxyFieldsType, StatusErrorType } from '../../../common/types';
+import ProxyFields from '../../../common/components/clusterConfiguration/ProxyFields';
+import UploadSSH from '../../../common/components/clusterConfiguration/UploadSSH';
+import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
+import DiscoveryImageTypeDropdown, { discoveryImageTypes } from './DiscoveryImageTypeDropdown';
+
+export type OcmDiscoveryImageFormValues = ImageCreateParams & ProxyFieldsType;
```

Comment:
This is a duplicate of https://github.com/openshift-assisted/assisted-ui-lib/blob/e0403af7397bcce17df95c8ce8a77f234147b9fc/src/common/components/clusterConfiguration/DiscoveryImageConfigForm.tsx#L51

Change this type to allow `'discovery-image-ipxe'` in `imageType` and use it in the OCM components.

Reply:
Thanks. I'll try it

Reply:
@jgyselov maybe I could change in the other PR that I have open. I've needed this PR to finish the other about trusted certificates.

Reply:
Same as before, it this can be fixed now, let's do it and then we can continue with the other task and keep it isolated to trust certificates.

Reply:
Changes done